src/expandableCalendar/WeekCalendar/presenter.spec.js

```diff
@@ -0,0 +1,359 @@
+const {UPDATE_SOURCES} = require('../commons');
+const {default: Presenter} = require('./presenter');
+
+describe('WeekCalendar presenter tests', () => {
+  const twoDaysSameWeek = {
+    date: '2021-01-01',
+    prevDate: '2021-01-02'
+  };
+
+  const twoDaysDifferentWeek = {
+    date: '2021-01-01',
+    prevDate: '2021-01-05'
+  };
+
+  const twoSomeDays = {
```

Comment:
I don't understand why you need this const? Isn't `twoDaysSameWeek` have the same principle?